static/js/theme-map/VerticalFullPageMapOrchestrator.js

```diff
@@ -373,6 +378,15 @@ class VerticalFullPageMapOrchestrator extends ANSWERS.Component {
    * Search the area or show the search the area button according to configurable logic
    */
   handleMapCenterChange () {
+    const isIframe = 'parentIFrame' in window;
+    const hasMapMoved = this.hasMapMoved;
```

Comment:
I think this logic is a bit confusing. As someone reading this method, my first inclination would be "why are they looking at the historical value of `hasMapMoved`, it's moving now!" It might be clearer if instead of keeping track of `hasMapMoved` we determine if this is the first invocation of the method.

Reply:
Yeah, that sounds good.